alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -90,6 +90,10 @@ pub type ConptyHandle = Arc<Conpty>;
 
 impl Drop for Conpty {
     fn drop(&mut self) {
+        // Caution! This will block until the conout pipe is drained. May cause deadlocks if the
+        // conout pipe has already been dropped by this point.
+        //
+        // See PR #3084 and https://docs.microsoft.com/en-us/windows/console/closepseudoconsole
```

Comment:
Many text editors like vim already recognize things like `XXX` and highlight it. Using `Caution!` here will likely make this look more like a normal comment than something important, so I'd recommend replacing `Caution!` with `XXX:`.

The phrasing `May cause deadlocks ...` is also kinda incorrect. It would be more accurate to say `Will cause deadlock ...`.